subscription/update_my_subscription_data.go

```diff
@@ -0,0 +1,32 @@
+// Package subscription implements the Subscription NEX protocol
+package subscription
+
+import (
+	"fmt"
+
+	nex "github.com/PretendoNetwork/nex-go"
+	"github.com/PretendoNetwork/nex-protocols-go/globals"
+)
+
+// UpdateMySubscriptionData sets the UpdateMySubscriptionData handler function
+func (protocol *SubscriptionProtocol) UpdateMySubscriptionData(handler func(err error, client *nex.Client, callID uint32, content []byte)) {
+	protocol.UpdateMySubscriptionDataHandler = handler
+}
+
+func (protocol *SubscriptionProtocol) handleUpdateMySubscriptionData(packet nex.PacketInterface) {
+	if protocol.UpdateMySubscriptionDataHandler == nil {
+		fmt.Println("[Warning] SubscriptionProtocol::UpdateMySubscriptionData not implemented")
+		go globals.RespondError(packet, SubscriptionProtocolID, nex.Errors.Core.NotImplemented)
+		return
+	}
+
+	client := packet.Sender()
+	request := packet.RMCRequest()
+
+	callID := request.CallID()
+	parameters := request.Parameters()
+
+	content := parameters[4:]
```

Comment:
Adding on to what @DaniElectra said, even if this is unknown it would be nice to still make it available in the handler. Since this is 4 bytes, even just treating it as a uint32 would suffice

Ideally all data sent in the request should be available in the handler